src/_guides/language/sound-dart.md

```diff
@@ -601,31 +620,9 @@ In the following example, you can substitute
 `List<MaineCoon>` is a subtype of `List<Cat>`.
 
 {:.passes-sa}
+<?code-excerpt "strong/lib/strong_analysis.dart (generic-type-assignment-MaineCoon)" replace="/MaineCoon/[!$&!]/g"?>
 {% prettify dart %}
-class Animal {
-  void feed() {}
-}
-
-class Cat extends Animal {}
-
-class MaineCoon extends Cat {}
-
-void feedAnimals(Iterable<Animal> animals) {
-  for (var animal in animals) {
-    animal.feed();
-  }
-}
-
-main(List<String> args) {
-  // Was: List<Cat> myCats = new List<Cat>();
-  List<Cat> myCats = new List<[!MaineCoon!]>();
-  Cat muffin = new Cat();
-  Cat winky = new Cat();
-  Cat bongo = new Cat();
-  myCats.addAll([muffin, winky, bongo]);
-
-  feedAnimals(myCats);
-}
+List<Cat> myCats = new List<[!MaineCoon!]>();
```

Comment:
Before this, please add the comment that was there before:

```
  // Was: List<Cat> myCats = new List<Cat>();
```

Reply:
Sure but I don't think that it is relevant to the presentation. What is being contrasted in the code excerpt is the type argument to `List` on the LHS vs. the type argument to `List` on the RHS.

I'll leave out the comment for now and I'll revisit later if you feel the need.

Reply:
The text "In the following example, you can substitute new List<Cat>() with new List<MaineCoon>()" refers to code (new List<Cat>()) that I actually looked above to find, since it seemed to be saying that I'd seen that code before.

Reply:
Ah, you are right. #607 fixes that. (Given that #607 is a minor fix to this approved PR, I'll assume that it is ok to merge it once Travis is green.)